src/components/AvatarIconType.ts

```diff
@@ -0,0 +1,13 @@
+import CONST from '@src/CONST';
+
+type AvatarFunction = () => void;
+
+type AvatarIcon = {
```

Comment:
Define these types directly in `MentionSuggestion` file, we can move it to a separate file as a follow-up.

Reply:
moved